Podfile

```diff
@@ -6,7 +6,7 @@ use_frameworks!
 target 'AWSAppSync' do
   pod 'AWSCore', '~> 2.9.0'
   pod 'SQLite.swift', '0.11.5'
-  pod 'ReachabilitySwift', '~> 4.3.0'
+  pod 'ReachabilitySwift', '4.3.0'
```

Comment:
Please add a comment explaining why we pinned this version

Reply:
Will add.